install/helm/agones/defaultfeaturegates.yaml

```diff
@@ -20,12 +20,12 @@ PodHostname: true
 ResetMetricsOnDelete: true
 SDKGracefulTermination: true
 StateAllocationFilter: true
+SplitControllerAndExtensions: true
```

Comment:
These are alphabetical, so it should move up one line.

Reply:
Done